Sources/PackageModel/PackageIdentity.swift

```diff
@@ -169,6 +175,22 @@ struct LegacyPackageIdentity: PackageIdentityProvider, Equatable {
 ///   ```
 ///   file:///Users/mona/LinkedList → /Users/mona/LinkedList
 ///   ```
+/// * Normalizing Windows local file system paths, when applicable:
+///   ```
+///   c:\user\mona\LinkedList → /user/mona/LinkedList
```

Comment:
Drive No. should be kept to ensure its uniqueness, I’d rather suggest `/c/user/Mona/LinkedList` here.